test/rhsmlib/services/test_products.py

```diff
@@ -72,7 +72,6 @@
         "capabilities": [],
         "hypervisorId": None,
         "contentTags": [],
-        "autoheal": True,
```

Comment:
ditto

Reply:
Reverted.

Reply:
Interestingly, I forgot to remove the same value in the next JSON object `CONTENT_JSON` just a few lines below.